src/Jh/DataImportMagento/Factory/ShipmentWriterFactory.php

```diff
@@ -20,11 +20,17 @@ public function __invoke(LoggerInterface $logger)
     {
         $orderModel           = \Mage::getModel('sales/order');
         $transaction          = \Mage::getModel('core/resource_transaction');
-
+        $trackingModel        = \Mage::getModel('sales/order_shipment_track');
+        $emailFlag            = \Mage::getStoreConfig('sales_email/shipment/enabled');
+        $options              = [
```

Comment:
can we inline this and cast it: `$options              = ['send_shipment_email' => (bool) \Mage::getStoreConfig('sales_email/shipment/enabled')];`

Also, update the option name, all the other options in the lib use snake case + `send_shipment_email` is a bit more descriptive and is used in one of the other writers.